src/source/raster_dem_tile_worker_source.js

```diff
@@ -21,11 +21,12 @@ class RasterDEMTileWorkerSource {
     }
 
     loadTile(params: WorkerDEMTileParameters, callback: WorkerDEMTileCallback) {
-        const uid = params.uid;
+        const uid = params.uid,
+            encoding = params.encoding || "mapbox";
```

Comment:
the `|| "mapbox"` shouldn't be needed since `WorkerDemTileParameters.encoding` is required.